src/sidebar/components/test/MenuArrow-test.js

```diff
@@ -0,0 +1,31 @@
+import { mount } from 'enzyme';
+
+import MenuArrow, { $imports } from '../MenuArrow';
+
+import { mockImportedComponents } from '../../../test-util/mock-imported-components';
```

Comment:
I notice our components are inconsistent about how they order the local vs test imports. I think the simplest thing would be to follow the more recent convention of having just two groups: one for third-party and one for local imports, and within the groups, sort by import path. I can look at standardizing this via a tool at some point.